src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<div class="borrower-info-body loan-use">
+		<span>
+			A loan of {{ amount | numeral('$0,0') }} {{ helpedLanguage }}
+			{{ borrowerCountLanguage }} {{ shortenedLoanUse }}
+		</span>
+		<router-link
+			:to="`/lend/${loanId}`"
+			v-kv-track-event="['Lending', 'click-Read more', 'Read more', loanId, 'true']">
+			<span
+				@click="$emit('track-loan-card-interaction', {
+					interactionType: 'viewBorrowerPage',
+					interactionElement: 'readMore'
+				})"
+			>Read more</span>
+		</router-link>
+		<!--
+        <div v-if="activeSort === 'loanLength'" class="loan-length">
+            <strong>Loan length:</strong> {{ loanLength }} months
+        </div>
+        -->
```

Comment:
It appears this has been deprecated

Reply:
I think we will need something like this to activate when on the popularity/trending now sort via Algolia. Good to leave commented out for now.